javascripts/src/dataclips.js

```diff
@@ -194,14 +202,28 @@ export default class Dataclips {
                 metadata: { style: xlsx_number_formats.time_formatter.id },
               };
             case "datetime":
-              return {
-                value: 25569 + (value.ts + value.offset * minMs) / dayMs,
-                metadata: { style: xlsx_number_formats.datetime_formatter.id },
-              };
+              if (disableSeconds) {
+                return new Date(value.ts).toLocaleString([], {year: 'numeric', month: 'numeric', day: 'numeric', hour: '2-digit', minute: '2-digit'})
+              } else {
+                return {
+                  value: 25569 + (value.ts + value.offset * minMs) / dayMs,
+                  metadata: { style: xlsx_number_formats.time_formatter.id },
+                };
+              }
             case "duration":
-              return {
-                value: value.as("day"),
-                metadata: { style: xlsx_number_formats.duration_formatter.id },
+              if (disableSeconds) {
+                const correctedHoursValue = value.values["days"] !== undefined ? value.values["hours"] : value.values["days"] * 24 + value.values["hours"]
+                if (value.values["days"] !== undefined) {
```

Comment:
what is this for ?? you have already calculated hours one line above. As for me this 'if' is not necessary and should be removed

Reply:
Thanks to that we get 93:23:45 instead of 3:21:23:45 (it's example)

Reply:
93:23:45 what is the value of "correctHoursValue' , isn't it already 93 ?? 
you perform the same manipulations twice